usr/share/rear/conf/default.conf

```diff
@@ -2797,16 +2815,21 @@ BEXTRACT_VOLUME=
 # name of a tape device or a disk block device as configured for bareos-sd.
 #  eg. Ultrium-1 or /dev/sda1
 BEXTRACT_DEVICE=
-# If you have more than one restore job defined for a client, define it as
-# BAREOS_RESTORE_JOB=client-restore
-# You can leave it unset as ReaR will tell you when it is needed
-# BAREOS_RESTORE_JOB=
-#
-# If you have more than one fileset configured for your client, specify one here.
-# Otherwise interaction is needed during restore.
-# Leave unset, if you have only one fileset for your client (this is almost always the case)
-# E.g. BAREOS_FILESET=Full
-# BAREOS_FILESET=
+#
+# Settings for BAREOS bconsole mode
+#
+# Bareos client name, if it differs from <hostname>-fd
+# E.g. BAREOS_CLIENT="$HOSTNAME-fd"
+BAREOS_CLIENT=
```

Comment:
If that is the sane default then I'd suggest setting it here.

Reply:
@joergsteffens Now you dynamically set this `$HOSTNAME-fd` in code.

Wouldn't it be simpler to set it here as a default and be happy if it is in the list of active clients? In the case of somebody overriding it manually nothing would change and in the case of a mismatch it would still fail.

Reply:
If you have configured your Bareos environment well, bconsole will only see 1 client. Currently the code just picks the one client. I wonder how the script should behave, when there is only 1 client, but the name differs from the default `$HOSTNAME-fd`. Currently it will pick the one client. But when BAREOS_CLIENT is set, it would have to overwrite this default, even independent of if it is set in default.conf or local/site.conf. 

For this reason, I've chosen this approach.
Having said that, there might also be environments, where all clients really match `$HOSTNAME-fd`, so using this also make sense.